fastlane/lanes/release.rb

```diff
@@ -504,19 +504,29 @@
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
-  build_url = buildkite_trigger_build(
-    buildkite_organization: BUILDKITE_ORGANIZATION,
-    buildkite_pipeline: BUILDKITE_PIPELINE,
-    branch: branch,
-    environment: { BETA_RELEASE: beta },
-    pipeline_file: 'release-builds.yml',
-    message: beta ? 'Beta Builds' : 'Release Builds'
-  )
+  environment = {
+    IS_BETA_RELEASE: beta,
```

Comment:
Just curious: Is the change from `BETA_RELEASE` to `IS_BETA_RELEASE` just polish you introduced or is there a wider discussion about it that I missed?

I don't mind either way, and I saw the only other occurrence in `release-build.yml` was updated 👌

Reply:
I think on WCiOS, this flag was named `BUILDKITE_BETA_RELEASE`, without really needing the `BUILDKITE_` prefix. So I renamed it to `IS_BETA_RELEASE` on that case. For consistency, I've also renamed this one to `IS_BETA_RELEASE`.

Reply:
FWIW if we're  gonna decide on a convention across our different repos on this, I'd vote for the env var to be named `IS_BETA_RELEASE` over just `BETA_RELEASE` to make it clear this is expected to be a boolean (and not e.g. the beta release version number as one might think when just reading the file)